sql/core/src/main/scala/org/apache/spark/sql/parquet/ParquetTableSupport.scala

```diff
@@ -105,8 +104,7 @@ private[parquet] class RowReadSupport extends ReadSupport[InternalRow] with Logg
       // If the parquet file is thrift derived, there is a good chance that
       // it will have the thrift class in metadata.
       val isThriftDerived = keyValueMetaData.keySet().contains("thrift.class")
-      parquetSchema = ParquetTypesConverter
-        .convertFromAttributes(requestedAttributes, isThriftDerived)
+      parquetSchema = ParquetTypesConverter.convertFromAttributes(requestedAttributes)
```

Comment:
It's not needed anymore, since the converter already handles parquet-thrift files in a consistent way via backwards-compatibility rules. It's also proven by `ParquetThriftCompatibilitySuite`.

(Hm, forgot to remove Thrift related code here.)